routes/quizzes.js

```diff
@@ -1,12 +1,15 @@
 const express = require('express');
 const router = express.Router();
 const Quiz = require('../models/quiz');
+const shuffle = require('../helpers/shuffle');
 
 //Getting all questions
 router.get('/', async (req, res) => {
   const { quizName } = req.query;
   try {
     const quizQuestions = await Quiz.find({ name: quizName });
+    quizQuestions.forEach(question => shuffle(question.answers));
+    shuffle(quizQuestions);
```

Comment:
```suggestion
    shuffle(quizQuestions);
```
It would be nice to be able to just call `shuffle` to shuffle the questions, without needing to know to do a `forEach` before. How about we move the `forEach` into `shuffle.js`?

Reply:
@abdulkadiret We're moving line 11 to `shuffle.js`, so we would change `shuffle.js` to something like:

```
const shuffleQuestion = arr => {
  for (let i = arr.length - 1; i > 0; i--) {
    const j = Math.floor(Math.random() * (i + 1));
    [arr[i], arr[j]] = [arr[j], arr[i]];
  }
  return arr;
};

const shuffle = quizQuestions =>
  quizQuestions.forEach(question => shuffleQuestion(question.answers));
```
This lets us pass in `quizQuestions` to the `shuffle` function and the function will shuffle everything for us. I haven't tested this code so please check that it works

Reply:
I changed the `shuffle.js` codes with above code and I removed line 11 from `routes/quizzes.js`. Unfortunately, it only shuffles the answers not the questions.

Reply:
Okay sure, thanks for giving it a go 👍